tests/constructor-basic/constructor-basic.variables.js

```diff
@@ -0,0 +1,299 @@
+const { gql } = require('@apollo/client');
+const { setupApp } = require('../helper-functions');
+let operations;
+
+const badConstructorBasicID = '6009dcd5f9855555907ebf5e';
+const newColor = {
+  name: [
+    { lang: 'ua', value: 'Світsadas' },
+    { lang: 'en', value: 'blackdasdas' },
+  ],
+  colorHex: '#f47ac6',
+  simpleName: [
+    { lang: 'ua', value: 'Чорний' },
+    { lang: 'en', value: 'black' },
+  ],
+};
+const newConstructorBasic = (materialID, colorID) => ({
+  name: [
+    { lang: 'ua', value: 'варіант 1' },
+    { lang: 'en', value: 'variant 1' },
+  ],
+  material: materialID,
+  color: colorID,
+  image: '/imageURL',
+  basePrice: 0,
+  available: true,
+  default: false,
+});
+const newConstructorBasicUpdateInp = (materialID, colorID) => ({
+  name: [
+    { lang: 'ua', value: 'варіант 5' },
+    { lang: 'en', value: 'variant 5' },
+  ],
+  material: materialID,
+  color: colorID,
+  image: '/imageURLkk',
+  basePrice: 0,
+  available: true,
+  default: false,
+});
+const newMaterial = colorID => ({
+  name: [
+    { lang: 'ua', value: 'тканина кордура' },
+    { lang: 'en', value: 'Cordura fabric' },
+  ],
+  description: [
+    { lang: 'ua', value: 'описання' },
+    { lang: 'en', value: 'some description' },
+  ],
+  purpose: 'some purpose',
+  colors: colorID,
+  available: true,
+  additionalPrice: 0,
+});
+
+const getConstructorData = construrtorBasic => ({
+  name: [
+    {
+      lang: construrtorBasic.name[0].lang,
+      value: construrtorBasic.name[0].value,
+    },
+    {
+      lang: construrtorBasic.name[1].lang,
+      value: construrtorBasic.name[1].value,
+    },
+  ],
+  material: { _id: construrtorBasic.material },
+  color: { _id: construrtorBasic.color },
+  image: construrtorBasic.image,
+  available: construrtorBasic.available,
+  default: construrtorBasic.default,
+});
+const getConstructorDataBeforeUpt = construrtorBasic => ({
+  name: [
+    {
+      lang: construrtorBasic.name[0].lang,
+      value: 'Назва до оновлення',
+    },
+    {
+      lang: construrtorBasic.name[1].lang,
+      value: 'Before update name',
+    },
+  ],
+  material: construrtorBasic.material,
+  color: construrtorBasic.color,
+  image: construrtorBasic.image,
+  available: false,
+  default: false,
+});
+const getConstructorDataForUpt = construrtorBasic => ({
+  name: [
+    {
+      lang: 'ua',
+      value: 'ПІСЛЯ',
+    },
+    {
+      lang: 'en',
+      value: 'After',
+    },
+  ],
+  material: construrtorBasic.material,
+  color: construrtorBasic.color,
+  image: '/new img',
+  available: true,
+  default: false,
+});
+const getConstructorDataForUptCompare = construrtorBasic => ({
+  _id: construrtorBasic._id,
+  available: construrtorBasic.available,
+  color: construrtorBasic.color._id,
+  default: construrtorBasic.default,
+  material: construrtorBasic.material._id,
+  image: construrtorBasic.image,
+  name: construrtorBasic.name,
+});
+const createColorMy = async newColor => {
```

Comment:
We already have this helper function in `material.variables.js`

Reply:
imporn createColor from helper function in material.variables.js